source/geometry_model/spherical_shell.cc

```diff
@@ -403,6 +404,9 @@ namespace aspect
 
           for (auto &connected_position: connected_positions)
             connected_position = rotation_matrix * connected_position;
+
+          for (auto &connected_velocity: connected_velocities)
+            connected_velocity = rotation_matrix * connected_velocity;
```

Comment:
This is interesting: The movement of points is the same operation as the rotation of the velocities. Is this always the case for arbitrary opening angles of the sector, or is this a coincidence?

Reply:
As far as I can tell by intuition it should always be the same transformation independent of opening angle. 
I suspect it is some underlying property of the coordinate transformation, but I did not write it down (likely related to the derivatives of the coordinate axes with respect to each other).